test/widgets/app_test.dart

```diff
@@ -51,4 +54,113 @@ void main() {
       ]);
     });
   });
+
+  group('ChooseAccountPage', () {
+    Future<void> setupChooseAccountPage(WidgetTester tester, {
+      required List<Account>? accounts,
```

Comment:
nit: requiring the caller to explicitly pass this makes sense, but then it may as well just take a list:
```suggestion
      required List<Account> accounts,
```

so the caller explicitly says `[]`, instead of saying `null` and having that mean `[]`.

(In fact, now that I think of that, even in the version where this parameter was optional it probably should have been `List<Account> accounts = [],` rather than being nullable.)